contributingGuides/TS_STYLE.md

```diff
@@ -381,7 +365,7 @@ type Foo = {
 
 <a name="file-organization"></a><a name="1.15"></a>
 
-- [1.15](#file-organization) **File organization**: In modules with platform-specific implementations, create `types.ts` to define shared types. Import and use shared types in each platform specific files. Do not use [`satisfies` operator](#satisfies-operator) for platform-specific implementations, always define shared types that complies with all variants. 
+- [1.15](#file-organization) **File organization**: In modules with platform-specific implementations, create `types.ts` to define shared types. Import and use shared types in each platform specific files. Do not use [`satisfies` operator](#satisfies-operator) for platform-specific implementations, always define shared types that complies with all variants.
```

Comment:
Since this is the last hook we're adding for styles and the app is migrated, maybe we should add some DOCS on using the style hooks in this PR (or a follow up, either way)

Reply:
Let's do this in a follow up PR :)